### DISCUSSION TITLE
在内网开发环境下，使用web版sumi如何离线安装.vsix插件？

### DISCUSSION THREAD

**Discussion by JulyStarry** [opensumi/core#1830] [Q&A]
在内网开发环境下，使用web版sumi如何离线安装.vsix插件？

**Comment by yantze** [comment]
脚本这里可以参考一下， vsix 就是一个 zip 包
https://github.com/opensumi/core/blob/dd889243d86ef3a9cb9f29acadd9b806a16d3ab5/scripts/download.js#L164

**Reply by JulyStarry** [reply]
我是用docker部署的，那文件路径如何设置？？

**Reply by yantze** [reply]
什么文件路径？core/scripts/download.js 脚本里移除下载逻辑，然后指定本地路径就好了吧
